doc/configuration.md

```diff
@@ -160,6 +160,10 @@ _Note: all fields are optional unless otherwise marked_
 * **update**
     * **group** (string): the update group to follow. Most users will want one of: stable, beta, alpha.
     * **server** (string): the server to fetch updates from.
+    * **pcr_policy_server** (string): the server to revceive postede TPM PCR policy from.
```

Comment:
```suggestion
    * **pcr_policy_server** (string): the server to receive posted TPM PCR policy from.
```
but I wonder where you got this description from - I think the purpose is reverse ("the server to POST TPM PCR policy to")